src/components/tx-flow/flows/TokenTransfer/ReviewSpendingLimitTx.tsx

```diff
@@ -128,6 +136,8 @@ const ReviewSpendingLimitTx = ({
           <ErrorMessage error={submitError}>Error submitting the transaction. Please try again.</ErrorMessage>
         )}
 
+        {isRejectedByUser && <ErrorMessage>{`You've rejected the transaction.`}</ErrorMessage>}
```

Comment:
```suggestion
        {isRejectedByUser && <ErrorMessage>You&apos;ve rejected the transaction.</ErrorMessage>}
```

Reply:
This error message is defined in 4 places. I would suggest to extract it into a separate component like we do for some of the other errors like `NonOwnerError` to avoid duplication.